library/common/config/config.cc

```diff
@@ -455,6 +455,7 @@ const char* config_template = R"(
     upstream_connection_options: *upstream_opts
     circuit_breakers: *circuit_breakers_settings
     typed_extension_protocol_options: *base_protocol_options_defs
+#{admin_interface}
```

Comment:
This is a pattern we're actually looking to avoid introducing more of. Can we find an alternative closer to your original approach?

Reply:
Done